tests/database_connection/test_mongo_connect.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 Universität Tübingen, DKFZ and EMBL
```

Comment:
The test won't run on Github actions, because it needs a mongodb instance to perform. Thus, we moved it to its seperate directory on purpose, so it still will run locally, but not on github actions.

Reply:
That's not a problem, we can make it work on Github actions either through a mock of mongo or running a mongo container during the action, similar to this:
https://github.com/ghga-de/ghga-service-chassis-lib/blob/main/.github/workflows/unit_and_int_tests.yaml#L10-L14